src/test/java/com/wikia/webdriver/testcases/featuredvideo/FeaturedVideoDesktopTests.java

```diff
@@ -18,7 +18,7 @@
 
   @Test
   public void videoIsPresentOnArticle() {
-    FeaturedVideoComponentObject video = new FeaturedVideoComponentObject()
+    FeaturedVideoDesktopComponentObject video = new FeaturedVideoDesktopComponentObject()
         .setAutoplayCookie(false)
         .openWikiArticle("FeaturedVideo");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "FeaturedVideo" 12 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)